backend/code_coverage_backend/backend/__init__.py

```diff
@@ -20,7 +20,7 @@ def create_app():
     taskcluster.auth()
     taskcluster.load_secrets(
         os.environ.get("TASKCLUSTER_SECRET"),
-        code_coverage_backend.config.PROJECT_NAME,
+        prefixes=["common", "backend", "code-coverage-backend"],
```

Comment:
Why do we have `backend` and `code-coverage-backend`?

Reply:
I would like to switch to simpler project names in the secret now that we have one secret per "family". With that we can quietly move secrets from `code-coverage-backend` to `backend`

Reply:
OK.